akka-edge-docs/src/main/paradox/feature-summary.md

```diff
@@ -62,7 +62,7 @@ share the stream of events from the database and fan out to connected consumer s
 
 ### Edge is producer
 
-This case describe when the edge service is the producer of events.
+This cases describe when the edge service is the producer of events.
```

Comment:
Same fix as above?

```suggestion
This case describes when the edge service is the producer of events.
```

Or `These cases describe` for plural.

Reply:
thanks, s at wrong word